compysition/actors/__init__.py

```diff
@@ -41,7 +41,8 @@
 from .eventrouter import EventRouter, EventXMLFilter, EventFilter, HTTPMethodEventRouter, EventJSONFilter, EventXMLXpathsFilter, SimpleRouter
 from .eventattributemodifier import (EventAttributeModifier, HTTPStatusModifier, XpathEventAttributeModifier, EventAttributeLookupModifier,
                                      HTTPXpathEventAttributeModifier, JSONEventAttributeModifier, HTTPJSONAttributeModifier, ErrorEventAttributeModifier, 
-                                     XMLErrorEventAttributeModifier, XMLEventAttributeModifier, XMLEventAttributeLookupModifier)
+                                     XMLErrorEventAttributeModifier, XMLEventAttributeModifier, XMLEventAttributeLookupModifier,
+                                     EventAttributeDelete, EventDataEventAttributeModifier, EventDataStaticAttributeModifier)
```

Comment:
I was under the impression that we already had the functionality of `EventDataEventAttributeModifier` and `EventDataStaticAttributeModifier` in the classes `EventAttributeLookupModifier` and `EventAttributeModifier`, respectively. Is there a special case that I have overlooked?

Reply:
Hmm I think you're right, those were added in #2, here I was just adding in the import that got missed. @donvier something we're missing?